inc/zoo/swar/SWAR.h

```diff
@@ -70,6 +71,19 @@ struct SWAR {
 
     constexpr T value() const noexcept { return m_v; }
 
+   constexpr static T baseFromLaneLiterals(std::initializer_list<T> args) noexcept {
+       T result = 0;
+       for (auto arg: args) {
+           result = (result << NBits) | arg;
+       }
+       return result;
```

Comment:
This is big endian, have you thought about which one ought to be the "canonical" endianness for literals?

Reply:
I think it should read most significant on the left, least significant on the right, like it is now.